test-suite/coq-makefile/local-late-extension/Makefile.local-late

```diff
@@ -1,4 +1,4 @@
 MYVOFILES := $(VOFILES)
 
 printvo::
-	@echo '$(MYVOFILES)'
+	@echo '$(MYVOFILES)' > $@
```

Comment:
Maybe get rid of the `@` so it's easier to debug if needed?
```suggestion
	echo '$(MYVOFILES)' > $@
```

Reply:
it changes nothing though